server/src/resources/cart/cart.repository.ts

```diff
@@ -0,0 +1,27 @@
+import { UUID } from 'crypto';
+import { cartDb } from '../../database';
+import { Cart } from './cart.model';
+
+export const getCarts = async (): Promise<Cart[]> => {
+  const carts = await cartDb.getCarts();
+  return carts;
+};
+
+export const getCart = async (userId: UUID): Promise<Cart | undefined> => {
+  const cart = await cartDb.getCart(userId);
+  return cart;
+};
+
+export const createCart = async (cart: Cart): Promise<Cart> => {
+  const carts = await cartDb.createCart(cart);
+  return carts;
+};
+
+export const updateCart = async (cart: Cart): Promise<Cart | undefined> => {
```

Comment:
Returning undefined is very confusing. Please, return data in proper format or with correct 204 status (behavior should be consistent)